runcellpose.py

```diff
@@ -325,6 +325,20 @@ def visible_settings(self):
 
         return vis_settings
 
+    def validate_module(self, pipeline):
+        """If using custom model, validate the model file opens"""
+        if self.mode.value == 'custom':
+            model_file = self.model_file_name.value
+            model_directory = self.model_directory.get_absolute_path()
+            model_path = os.path.join(model_directory, model_file)
+            try:
+                open(model_path)

```

Comment:
This will test for absence of the file, but probably not function of the file. Depending on how intensive it is to try to actually run `model = models.CellposeModel(pretrained_model=model_path, gpu=self.use_gpu.value)` (and/or whether it's bad to do it more than once), it might be the least-worst thing to do, but just a thing to note